config/op-validator.toml

```diff
@@ -37,3 +37,6 @@ ccc
 [[result]]
   name = 'desafio-102'
   output = "foobar"
+[[result]]
+  name = 'desafio-13'
+  output = "ok"
```

Comment:
Talvez trocar o "ok" aqui por algo que torne mais claro que isso está sendo checado pelo código. Talvez "built-in" ou "hardwired", "code", etc? Ou talvez até o numero da função (tipo validKnightsD13).

Reply:
ok

Reply:
Alterei pra validKnightsD13.